lib/db_utils.go

```diff
@@ -202,10 +202,29 @@ var (
 	_PrefixRecloutedPostHashReclouterPubKeyRecloutPostHash = []byte{46}
 	_PrefixDiamondedPostHashDiamonderPKIDDiamondLevel      = []byte{47}
 
+	// Prefixes for NFT ownership entries:
+	// 	<prefix, NFTPostHash [32]byte, SerialNumber uint64> -> NFTEntry
+	_PrefixPostHashSerialNumberToNFTEntry = []byte{48}
+	//  <prefix, PublicKey [33]byte, IsForSale bool, BidAmountNanos uint64, NFTPostHash[32]byte, SerialNumber uint64> -> NFTEntry
+	_PrefixPublicKeyIsForSaleBidAmountNanosPostHashSerialNumberToNFTEntry = []byte{49}
+
+	// Prefixes for NFT bids:
+	_PrefixPostHashSerialNumberBidNanosBidderPKID   = []byte{50}
+	_PrefixBidderPKIDPostHashSerialNumberToBidNanos = []byte{51}
+
+	// <prefix, PublicKey [33]byte> -> uint64
+	_PrefixPublicKeyToBitCloutBalanceNanos = []byte{52}
+	// Block reward prefix (used for deducting immature block rewards from bitclout balances):
+	// <hash BlockHash> -> <pubKey [33]byte, uint64 blockRewardNanos>
+	_PrefixPublicKeyBlockHashToBlockReward = []byte{53}
+
+	// <prefix>, NFTPostHash [32]byte, SerialNumber uint64 -> []NFTBidEntry
+	_PrefixPostHashSerialNumberToAcceptedBidEntries = []byte{54}
```

Comment:
Add a comment explaining how there can be multiple accepted BidEntries. It wasn't immediately obvious to me why this needed to be a slice, and slices are unusual in this context so a better comment would help.

Reply:
I would add in the comment that this is not core to consensus, assuming that's the case. We don't want to unknowingly start relying on this since it could get inefficient.